crates/evm/evm/src/inspectors/stack.rs

```diff
@@ -396,16 +382,9 @@ impl<DB: DatabaseExt> Inspector<DB> for InspectorStack {
                 &mut self.printer
             ],
             |inspector| {
-                let status = inspector.step(interpreter, data);
-
-                // Allow inspectors to exit early
-                if status != InstructionResult::Continue {
```

Comment:
Interesting, how is this allowed now?

Reply:
it's handled outside of the inspector now, if it should revert, the interepreter's instruction_result can be updated directly